wrappers/desktop.go

```diff
@@ -163,6 +164,15 @@ func sanitizeDesktopFile(s *snap.Info, rawcontent []byte) []byte {
 	return []byte(strings.Join(newContent, "\n"))
 }
 
+func updateDesktopDatabase() error {
+	if _, err := exec.LookPath("update-desktop-database"); err == nil {
```

Comment:
Hm, my understanding is that update-desktop-database is a xdg thing, so we probably want to run it everywhere where it is available? Happy to run more and distro specific stuff once we have a use-case.